certify-service/src/main/java/io/mosip/certify/CertifyServiceApplication.java

```diff
@@ -8,12 +8,28 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
```

Comment:
I am not able to see above imports getting used

Reply:
removed these, as they were not being used anywhere